src/Rule/Sanitize/Regex.php

```diff
@@ -32,7 +34,7 @@ class Regex
      * @return bool True if the value was sanitized, false if not.
      *
      */
-    public function __invoke($subject, $field, $expr, $replace)
+    public function __invoke(object $subject, string $field, $expr, $replace): bool
```

Comment:
I am wondering whether we need to remove is_scalar check and allow array also to be passed to $exp and $replace .